src/pivot-table/components/cells/DimensionTitleCell.tsx

```diff
@@ -66,9 +64,11 @@ const DimensionTitleCell = ({
 
   const isDim = cell.id !== "PSEUDO-DIM";
 
-  const headerData = useMemo<Header>(
+  const headerCell = useMemo<HeaderCell>(
```

Comment:
Is there any difference between the `headerCell` and `cell` now?

Reply:
Does not appear so. Should be no need to create `headerCell``

Reply:
Done. No need to memoize anymore